Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apt-get update && apt-get install -y librocksdb-dev curl xxd openssl
 # Install chainweb applications
 WORKDIR /chainweb
 # RUN curl -Ls "https://github.com/kadena-io/chainweb-node/releases/download/<chaineweb-version>/<chainweb-binary-version>" | tar -xzC "/chainweb/"
-RUN curl -Ls "https://kadena-cabal-cache.s3.amazonaws.com/chainweb-node/chainweb.8.6.5.ubuntu-18.04.be3d78a.tar.gz" | tar -xzC "/chainweb/"
+RUN curl -Ls "https://kadena-cabal-cache.s3.amazonaws.com/chainweb-node/chainweb.8.6.5.ubuntu-18.04.6f29cef.tar.gz" | tar -xzC "/chainweb/"
```

Comment:
I think, we should use ghc-8.8.3 for this (it used to be 8.8.3). GHC-8.6.5 isn't supported any more. Actually, we should switch to 8.8.4 (but I don't think we have builds for it yet), which has some important bug fixes.